mesh_tensorflow/transformer/utils.py

```diff
@@ -1203,6 +1217,33 @@ def clean_decodes(ids, eos_id=1, pad_id=0, length_axis=-1):
   return tf.where_v2(valid_ids, ids, pad_id)
 
 
+def compute_score(logits, targets, model_type):
```

Comment:
Nitpick: Name `compute_scores` instead of `compute_score`

Reply:
Thanks @daphnei  for the review. I've updated the function name to compute_scores.